src/Common/src/TypeSystem/IL/Stubs/ArrayMethodILEmitter.cs

```diff
@@ -50,7 +50,7 @@ public MethodIL EmitIL()
                     break;
 
                 case ArrayMethodKind.Ctor:
-                    // .ctor is implemented as a JIT helper and the JIT shouldn't be asking for the IL.
```

Comment:
This comment explains why there's nothing in this case statement.

It's not a description of the default case (which is how it looks with the reformatting...).